thunder/torch/__init__.py

```diff
@@ -3005,6 +3005,52 @@ def topk(
     return clang.topk(a, k, dim, largest, sorted, out=out)
 
 
+@torchsymbol(torch.atleast_1d, is_method=True)
+def atleast_1d(
+    arg: Union[TensorLike, Sequence[TensorLike]], *args: TensorLike
+) -> Union[TensorLike, tuple[TensorLike, ...]]:
+    if not args and isinstance(arg, Sequence):
```

Comment:
If the inputs is just `*tensors`, then I think this `if` statement can be removed, and the function would just be:

```
res = tuple(a if a.ndim >= 1 else unsqueeze(a, 0) for a in args)
return res if len(res) > 1 else res[0]
```